src/helpers.test.js

```diff
@@ -6,95 +6,172 @@ import {
     getDefaultValueForType,
     getDefaultValueFromOptions,
     getFunctionName,
+    removeFirstKeyIfNumber,
+    getErrorIndexFromKeys,
 } from './helpers';
 
 describe('helpers', () => {
-    it('should create object from another object with specific keys', () => {
-        const bar = {
-            foo: 'bar',
-            foo2: 'bar2',
-            foo3: 'bar3',
-            foo4: 'bar4',
-        };
-        const foo = pick(bar, ['foo', 'foo2']);
+    describe('pick', () => {
+        it('should create object from another object with specific keys', () => {
```

Comment:
There should be a scenario when some property is picked, but **not** present in given object.

Reply:
added